### DISCUSSION TITLE
keepasxc support

### DISCUSSION THREAD

**Discussion by Eugene7219** [Alex313031/thorium#116] [Q&A]
Hi !!!
is this possible enable keepasxc extension support?
By default its not work with thorium.

Thanks!

**Comment by gz83** [comment]
@Eugene7219 

Hello, for questions about keepassxc, please refer to the link below

https://github.com/Alex313031/Thorium-Special/issues/36#issuecomment-1418687197

**Comment by Alex313031** [comment]
@Eugene7219 Yeah, following that comment (changing the dir to the actual location on your platform if you are not on MacOS) should work.

On linux this would be:
`~/.config/thorium`
On windows it would be:
`C:\Users\$USERNAME\AppData\Local\Thorium\User Data` substituting $USERNAME for your actual username
For the portable releases, obviously it will be in the USER_DATA dir side by side with the binary.

**Reply by Aldo-cav** [reply]
I can confirm that this works on MacOS too, I've installed Thorium via homebrew and my config location is:
`~/Library/Application Support/Thorium/NativeMessagingHosts`
